cmd/controller/app/options/options.go

```diff
@@ -156,7 +156,7 @@ const (
 	defaultMaxConcurrentChallenges   = 60
 
 	defaultPrometheusMetricsServerAddress = "0.0.0.0:9402"
-	defaultHealthzServerAddress           = "127.0.0.1:10257"
+	defaultHealthzServerAddress           = "0.0.0.0:10257"
```

Comment:
Why do we need to listen on 0? Kubelete should be able to probe on localhost and nothing else should need to access it by default? Especially given that this is http maybe would be better to restrict access?

Reply:
That's what I originally thought too, but it failed when I tested on GKE:
 * https://github.com/cert-manager/cert-manager/pull/5962#discussion_r1177752166

My Kubernetes networking knowledge is rusty.